### PR TITLE
Vb diagnostic enum

### PR DESCRIPTION
**Pull request by jwooley**
Resolves Issue #244

### CODE REVIEW

**Comment by giggio** [comment]
Build is breaking: https://ci.appveyor.com/project/code-cracker/code-cracker/build/1.0.0.356

**Comment by jwooley** [comment]
Fixed

**Comment by giggio** [comment]
Build is now passing: https://ci.appveyor.com/project/code-cracker/code-cracker/build/1.0.0.357.

File: src/VisualBasic/CodeCracker/Performance/MakeLocalVariableConstWhenPossibleAnalyzer.vb [hunk 1]
```diff
@@ -3,26 +3,27 @@ Imports Microsoft.CodeAnalysis.Diagnostics
...
-            DiagnosticId,
+            Id,
             Title,
             MessageFormat,
             Category,
             DiagnosticSeverity.Info,
             isEnabledByDefault:=True,
             description:=Description,
-            helpLink:=HelpLink.ForDiagnostic(DiagnosticId))
+            helpLink:=HelpLink.ForDiagnostic(Id))
```
Comment (giggio) [inline src/VisualBasic/CodeCracker/Performance/MakeLocalVariableConstWhenPossibleAnalyzer.vb]: `HelpLink.ForDiagnostic` should be using the Enum, not a string.

**Comment by jwooley** [comment]
Updated to use enum for help link as well.

**Pull request merged**